src/solvers/dgsem_tree/dg_3d_parabolic.jl

```diff
@@ -597,12 +599,11 @@ function calc_boundary_flux_by_direction_divergence!(surface_flux_values::Abstra
 end
 
 function prolong2mortars!(cache,
-                          flux_viscous::Tuple{AbstractArray, AbstractArray,
-                                              AbstractArray},
+                          flux_viscous::Vector{Array{uEltype, 5}},
                           mesh::TreeMesh{3},
                           equations_parabolic::AbstractEquationsParabolic,
                           mortar_l2::LobattoLegendreMortarL2,
-                          surface_integral, dg::DGSEM)
+                          surface_integral, dg::DGSEM) where {uEltype <: Real}
```

Comment:
See above on `prolong2mortars` being called with `cache` and not `cache_parabolic`. Comments explaining why this is done would help.